extension/js/add_input_icon.js

```diff
@@ -210,15 +212,18 @@ async function addRelayIconToInput(emailInput) {
     const maxNumAliasesReached = numAliasesRemaining === 0;
     if (maxNumAliasesReached) {
       generateAliasBtn.disabled = true;
+      sendInPageEvent("viewed-menu", "input-menu-max-aliases-message")
```

Comment:
I also wondered about this. If we change the `click` actions to `clicked-something`, it makes for easier GA scanning, but then the `eventLabel` and `eventAction` have basically the same value.

Reply:
True - it could be that we just don't know the best values to use here because we haven't tried to report on them yet. Fine to leave as-is to avoid paralysis-by-analysis.